### DISCUSSION TITLE
FTX support

### DISCUSSION THREAD

**Discussion by jvirtanen** [paritytrading/philadelphia#223] [General]
Philadelphia now [supports FTX](https://github.com/paritytrading/philadelphia-extras/tree/master/libraries/ftx). I also wrote [a small example application](https://github.com/jvirtanen/ftx-fix-example) demonstrating the authentication process.

**Comment by zhuyanxuan** [comment]
why need [stunnel](https://www.stunnel.org/). Does anything instead of it in jdk?

**Reply by jvirtanen** [reply]
Philadelphia 2.0.0 (coming soon) will make it possible to use e.g. [TLS Channel](https://github.com/marianobarrios/tls-channel) directly in the JVM process.

**Reply by jvirtanen** [reply]
I just updated [the example application](https://github.com/jvirtanen/ftx-fix-example) to use TLS Channel instead of stunnel.
